.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
                 name: Run flake8
                 command: |
                     . venv/bin/activate
-                    flake8 --max-line-length=100 tests/ dash_bio/utils/ dash_bio/component_factory/
+                    flake8 --max-line-length=100 tests/ dash_bio/component_factory/
```

Comment:
good catch! Although I'm not sure why this wasn't causing the CI tests to fail, since the `dash_bio/utils` folder doesn't even exist any more.

Reply:
I don't think this was causing any CI test to fail... unless I missed something. The entire changeset is not about fixing, only cleaning up. Thanks for you review!